src/flash.c

```diff
@@ -134,6 +142,12 @@ pu_flash_class_init(PuFlashClass *class)
                             "Path to prefix all file URIs with in the layout configuration",
                             NULL,
                             G_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY);
+    props[PROP_SKIP_CHECKSUMS] =
+        g_param_spec_boolean("skip-checksums",
+                             "Modifier to skip checksums",
+                             "Modifier to skip checksum verification for all files when writing",
+                             FALSE,
```

Comment:
Set default value to FALSE, as NULL generates compiler warnings.

Reply:
Ah yes, that's correct, a boolean obviously needs `TRUE`/`FALSE` as default. Didn't run your changes yet, so this went unnoticed.